json-endpoints/jetpack/class.jetpack-json-api-sync-endpoint.php

```diff
@@ -189,6 +189,8 @@ protected function result() {
 			return new WP_Error( 'invalid_number_of_items', 'Number of items needs to be an integer that is larger than 0 and less then 100', 400 );
 		}
 
+		$number_of_items = intval( $args[ 'number_of_items' ] );
```

Comment:
Should `intval` be an `absint` instead? I'm doubtful we'll want to allow negative integers here.

Reply:
TIL `absint` :) Thanks!